dissect/target/plugins/os/windows/log/etl.py

```diff
@@ -31,12 +31,15 @@ def _build_record(self, etl_event: Event, etl_path: str, target: Target):
         record_values["_target"] = target
 
         for key, value in etl_event.event_values().items():
+            record_type = "bytes"
             if isinstance(value, list):
-                record_fields.append(("string[]", key))
+                record_type = "string[]"
             elif isinstance(value, int):
-                record_fields.append(("varint", key))
-            else:
-                record_fields.append(("string", key))
+                record_type = "varint"
+            elif isinstance(value, str) and value.isprintable() or isinstance(value, Instance):
```

Comment:
If I understand the etl code correct, `etl_event.event_values()` are either key,value pairs from normal Python `dict()`s (from the headers `additional_header_fields()`) or cstruct `Instance._values` key,value pairs (from the parsed structure).

In that light I don't see how a value that is a `str` instance can ever not be a `"string"`. So I don't see why the `is_printable()` is needed. Also classifying any value that itself is a cstruct `Instance` as a `"string"` seems strange. Could you explain?

Finally is the precendece correct or should there be some brackets like:
```suggestion
            elif (isinstance(value, str) and value.isprintable()) or isinstance(value, Instance):
```